src/scripts/client/modules/gradebook.js

```diff
@@ -373,11 +373,14 @@
                       }
                   }
                 } else {
-                  console.log("Submission from other path", submission, students[ studentId ])
+                  console.log('Submission from other path', submission, students[ studentId ]);
                 }
             } );
 
         }
+        if (sessionData.students.length === 0) {
+          throw new Error('no students');
```

Comment:
Why are we `throw`ing here? Will this ever get caught? If we're justing kicking out from the function should we just `return;`?